hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/views/missingContainers/missingContainers.tsx

```diff
@@ -362,16 +362,16 @@ export class MissingContainers extends React.Component<Record<string, object>, I
         </div>
         <div className='content-div'>
           <Tabs defaultActiveKey='1'>
-            <TabPane key='1' tab="Missing">
+            <TabPane key='1' tab={(missingDataSource && missingDataSource.length > 0) ? `Missing (${missingDataSource.length})` : 'Missing'}>
```

Comment:
Can we extract the `Missing` label out of the condition? something like this?:
```suggestion
            <TabPane key='1' tab='Missing'{(missingDataSource && missingDataSource.length > 0) ? ` (${missingDataSource.length})` : ''}>
```

Reply:
Completed Changes Suggested by Siyao to avoid code duplication.